code/__HELPERS/reagents.dm

```diff
@@ -72,3 +72,24 @@
 	if(!GLOB.chemical_reactions_list[primary_reagent])
 		GLOB.chemical_reactions_list[primary_reagent] = list()
 	GLOB.chemical_reactions_list[primary_reagent] += R
+
+/proc/choose_reagent_id(mob/user)
+	var/chosen_id
+	switch(alert(user, "Choose a method.", "Add Reagents", "Search", "Choose from a list"))
```

Comment:
You just removed the "I'm feeling lucky" option.

Reply:
fuck you're right